mono/tests/Makefile.am

```diff
@@ -353,7 +353,6 @@ BASE_TEST_CS_SRC=		\
 	bug-340662_bug.cs	\
 	bug-325283.2.cs	\
 	thunks.cs \
-	winx64structs.cs \
 	nullable_boxing.2.cs	\
```

Comment:
It's moved to the profile_disabled tests because it causes a crash inside the LLVM verifier.